features/old/api/features.feature

```diff
@@ -8,59 +8,80 @@ Feature: Features :)
     And an application plan "Basic" of provider "foo.3scale.localhost"
     Given current domain is the admin domain of provider "foo.3scale.localhost"
 
-  @javascript @wip
+  @javascript
   Scenario: Create a feature
     When I log in as provider "foo.3scale.localhost"
     And I go to the edit page for plan "Basic"
     And I follow "New feature"
     And I fill in "Name" with "Free T-shirt"
+    And I fill in "System name" with "tee"
     And I fill in "Description" with "T-shirt with logo of our company for free."
-    And I press "Create Feature"
-    Then I should see enabled feature "Free T-shirt"
-    And provider "foo.3scale.localhost" should have feature "Free T-shirt"
-    And feature "Free T-shirt" should be enabled for plan "Basic"
+    And I press "Save"
+    Then I should see "T-shirt with logo of our company for free."
 
-  # TODO: Test that the "No features yet" notice dissapears when first feature is created,
-  #       and appears when last one is deleted.
+  @javascript
+  Scenario: "No features yet" notice disappears when first feature is created
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
```

Comment:
As I mentioned previously, we don't need a service plan. The features behavior is pretty much the same. We already have an **application** plan called "Basic", defined in the scenario background, so there is absolutely no need to create another **service** plan (which also requires some switches enabling), because not only it does not help, but it also can make the things more confusing:

- A question may rise: why are we suddenly switching to service plans, if there is an application plan in the background and in the first test
- Two plans with the same name "Basic" may also get confused if some steps do not take the type of plan into account